cmd/containerd-stargz-grpc/main.go

```diff
@@ -57,21 +59,30 @@ const (
 	defaultLogLevel            = logrus.InfoLevel
 	defaultRootDir             = "/var/lib/containerd-stargz-grpc"
 	defaultImageServiceAddress = "/run/containerd/containerd.sock"
+	defaultFuseManagerAddress  = "/run/containerd-stargz-grpc/stargz-fuse-namanger.sock"
+	defaultFuseManagerPath     = "/usr/bin/stargz-fuse-manager"
```

Comment:
This should be `/usr/loca/bin/stargz-fuse-manager` because this is the default install location of our `make install`.